config/jobs/kubernetes/sig-k8s-infra/oci-proxy/sig-k8s-infra-oci-proxy-presubmits.yaml

```diff
@@ -0,0 +1,46 @@
+presubmits:
+  kubernetes-sigs/oci-proxy:
+  - name: pull-oci-proxy-verify
+    decorate: true
+    always_run: true
+    optional: true
+    spec:
+      containers:
+      - image: golang
```

Comment:
these just use golang as a known image with make / bash, if the golang version matches we will automatically use it but if it doesn't we will fetch it with gimme in the make system anyhow.

same for the other jobs except shellcheck